app/locations/index.js

```diff
@@ -9,17 +9,18 @@ const {
   setUserLocations,
   checkLocationsLength,
   setLocation,
+  setRegion,
   setAllLocations,
   setRegions,
 } = require('./middleware')
 
 router.use(setUserLocations)
 
 // Define routes
-// todo apply only to PMU journeys
```

Comment:
On a note for this, we may need to add another permissions to restrict the region routes and the display of them in the view behind that permission

Reply:
You mean the `/region/:regionId` route?

Reply:
Yes, sorry, that's the route I meant. I suppose it depends how strict we want to be from an application perspective.

If the logic is handled in the view where the user would never see the link, how much do we feel we want to protect the route? In this instance, if a user goes to it manually the outcome doesn't feel as bad as if a user was trying to access a resource that they shouldn't have permission to do.

Reply:
With the current code, you will be able to retrieve all moves for that region.
I'm not sure how the backend works — does it check the permissions for a user?
If it doesn't, anyone can use the full region dashboard as long as they know the ID of the region.